src/calculateRentalCost.js

```diff
@@ -29,6 +29,19 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const sum = days * 40;
+  const smallDiscount = 20;
+  const bigDiscount = 50;
+
+  if (days > 6) {
+    return sum - bigDiscount;
+  } else {
+    if (days > 2) {
+      return sum - smallDiscount;
+    }
+  }
```

Comment:
else is redundant here (point 3 in checklist)
```suggestion
  if (days > 6) {
    return sum - bigDiscount;
  }
  
  if (days > 2) {
    return sum - smallDiscount;
  }
  
```